src/main/include/subsystems/Shooter.h

```diff
@@ -44,4 +46,6 @@ class Shooter : public frc2::SubsystemBase {
 
   frc::DoubleSolenoid& mhood;
   frc::DoubleSolenoid& mlatch;
+
+  double p, i, d, f;
```

Comment:
pidf are all constants. Put these in the .cpp at the top in the `namespace {` block.